### PR TITLE
Fix example file generation script to adapt for macOS

### PR DESCRIPTION
**Pull request by cblp**
Must work on Linux, too, due to BSD compatibility

### CODE REVIEW

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
If you want to make the `mkfiles.sh` script macOS compatible then you need to utilize `uname` to do the correct thing given a particular os.

**Comment by cblp** [comment]
> If you want to make the `mkfiles.sh` script macOS compatible then you need to utilize `uname` to do the correct thing given a particular os.

@Jimbo4350 I think these changes are compatible with GNU/Linux, too, since GNU utilities are compatible with the BSD interface. I'll double-check.

**Comment by cblp** [comment]
It works correctly on both linux and macOS

File: scripts/byron-to-alonzo/mkfiles.sh [hunk 1]
```diff
@@ -312,7 +313,7 @@ cp ../configuration/cardano/shelley_qa-alonzo-genesis.json shelley/genesis.alonz
 # We're going to use really quick epochs (300 seconds), by using short slots 0.2s
 # and K=10, but we'll keep long KES periods so we don't have to bother
 # cycling KES keys
-sed -i shelley/genesis.spec.json \
+sed -i~ \
```
Comment (Jimbo4350) [inline scripts/byron-to-alonzo/mkfiles.sh:316]: This now creates two `genesis.spec.json` files which makes things ambiguous

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: scripts/byron-to-alonzo/mkfiles.sh (hunk 1 above)
Comment (cblp) [reply scripts/byron-to-alonzo/mkfiles.sh]: done

**Review by Jimbo4350 — APPROVED** [review]
Nice :+1:

**Comment by cblp** [comment]
bors merge

**Pull request merged**